sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosClient.java

```diff
@@ -31,7 +31,7 @@ public final class CosmosClient implements Closeable {
     private final CosmosAsyncClient asyncClientWrapper;
 
     CosmosClient(CosmosClientBuilder builder) {
-        this.asyncClientWrapper = builder.buildAsyncClient();
+        this.asyncClientWrapper = builder.buildAsyncClient(false);
```

Comment:
why not enabled for sync client?

Reply:
Because sync-client will also emit it in the buildClient() method